homeassistant/components/rejseplanen/entity.py

```diff
@@ -0,0 +1,45 @@
+"""Updater status sensor for Rejseplanen integration."""
+
+from typing import Any
+
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+
+
+class RejseplanenUpdaterStatusSensor(CoordinatorEntity):
```

Comment:
```suggestion
class RejseplanenUpdaterStatusSensor(CoordinatorEntityp[..]):
```
Please add the coordinator type here